doc/instances.md

```diff
@@ -666,6 +667,22 @@ proxy\_protocol | bool      | false         | no        | Whether to use the HAP
 security.uid    | int       | 0             | no        | What UID to drop privilege to
 security.gid    | int       | 0             | no        | What GID to drop privilege to
 
+### Type: unix-hotplug
+Unix hotplug device entries make the requested unix device
+appear in the container's `/dev` and allow read/write operations to it
+if the device exists on the host system.
+
+The following properties exist:
+
+Key         | Type      | Default           | Required  | Description
+:--         | :--       | :--               | :--       | :--
+vendorid    | string    | -                 | no        | The vendor id of the USB device.
+productid   | string    | -                 | no        | The product id of the USB device.
```

Comment:
The description shouldn't end with a period

Reply:
Also, it's saying `USB device` which isn't necessarily true.

Reply:
And you should probably mention that at least one of vendorid or productid needs to be specified.